frontend/src/metabase/dashboard/components/DashboardActions.jsx

```diff
@@ -78,32 +73,7 @@ export const getDashboardActions = (
     }
 
     if (canShareDashboard) {
-      buttons.push(
-        <DashboardSharingEmbeddingModal
-          key="dashboard-embed"
-          additionalClickActions={() => self.refs.popover.close()}
-          dashboard={dashboard}
-          enabled={
-            !isEditing &&
-            !isFullscreen &&
-            ((isPublicLinksEnabled && (isAdmin || dashboard.public_uuid)) ||
-              (isEmbeddingEnabled && isAdmin))
-          }
-          isLinkEnabled={canShareDashboard}
-          linkText={
-            <Tooltip
-              isLinkEnabled={canShareDashboard}
-              tooltip={
```

Comment:
Since the rendering is based on `canShareDashboard` I'm fairly sure this tooltip can never be exposed